hunt/config/initializers/simple_form.rb

```diff
@@ -127,7 +127,7 @@
   # in this configuration, which is recommended due to some quirks from different browsers.
   # To stop SimpleForm from generating the novalidate option, enabling the HTML5 validations,
   # change this configuration to true.
-  config.browser_validations = false
+  config.browser_validations = true
```

Comment:
Oh cool! I wasn't aware of this config. This is definitely a valid option, but let's switch it back to false for now so that we can make use of the rails model validations/errors.